### ISSUE TITLE
Broken experiment descriptions shown during prod deploy

### ISSUE THREAD

**Issue by jaredhirsch** [mozilla/testpilot#2229]
Opened the prod site during a deploy and saw some broken descriptions for the new experiments:

<img width="1009" alt="screen shot 2017-02-22 at 11 10 59 am" src="https://cloud.githubusercontent.com/assets/96396/23228132/5bbab0c6-f8f0-11e6-893c-938e4986ba56.png">

Need to investigate, but seems like a stale cached .ftl file (no strings for the new experiments).

A hard refresh fixed the issue.

@lmorchard suggested putting assets in a hashed directory name, so that pages load a correct set of assets. Sounds good to me, might be an opportunity to take a cleanup pass at the build/deploy tools.

**Comment by SoftVision-CosminMuntean** [comment]
I think this issue is related to #2126. 
I have managed to reproduce this on Test Pilot website using a old Firefox profile with Test Pilot( v0.9.4-dev-045fca1) add-on installed. 
The new experiments pages are also affected, but after I refreshed the page the strings was correctly displayed.

![sonooze tabs](https://cloud.githubusercontent.com/assets/19226534/23249840/02f0a45e-f9af-11e6-98d9-11fcd64805e6.png)

Also, I noticed that the "New" badge from Test Pilot icon is red, but I am not sure why. Maybe this happens only on old profiles, I will investigate this.
